examples/backends/vllm/launch/disagg_omni_pd.sh

```
@@ -0,0 +1,106 @@
+#
+# 4-stage PD disaggregated Qwen2.5-Omni text-to-text+audio generation.
+# Stage 0: thinker_prefill (GPU 0) — processes prompt, exports KV via NixlConnector
+# Stage 1: thinker_decode  (GPU 1) — resumes from KV, generates tokens
+# Stage 2: talker           (GPU 1) — enhanced text generation (AR)
+# Stage 3: code2wav         (GPU 1) — audio generation
+# Router: orchestrates the 4-stage pipeline, formats response
+set -e
+trap 'echo Cleaning up...; kill 0' EXIT
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source "$SCRIPT_DIR/../../../common/launch_utils.sh"
```

Shellcheck (0.11.0)

[info] Not following: ./../../../common/launch_utils.sh was not specified as input (see shellcheck -x).

(SC1091)